tests/local.rs

```diff
@@ -230,7 +230,7 @@ async fn stale_element(c: Client, port: u16) -> Result<(), error::CmdError> {
     .await?;
 
     match elem.click().await {
-        Err(error::CmdError::NoSuchElement(_)) => Ok(()),
+        Err(error::CmdError::Standard(_)) => Ok(()),
```

Comment:
This change doesn't seem quite right?

Reply:
hmm, yeah that's not specific enough after the change.

However, this leads to a new issue. Bringing `NoSuchElement` under `CmdError::Standard` creates more verbosity when checking for errors like this, which I suspect was the motivation for splitting these common variants out in the first place.

I'd like to solve this for all variants so that this kind of code (which is extremely common in browser automation) can be nicer to write.

Something like this is just too cumbersome:
```rust
match elem.click().await {
    Err(CmdError::Standard(w)) if w.error == ErrorStatus::NoSuchElement => {..}
    _ => {..}
}
```

One option is to do what the AWS SDK does and provider helper methods for checking the error variant. I.e.
```rust
match elem.click().await {
    Err(e) if e.is_no_such_element() => {..}
    _ => {..}
}
```
It's still not as nice as a flat enum though :(

Any other suggestions?

Reply:
actually this particular test expects a stale element but the point remains the same

Reply:
Yep, that was indeed the original motivation :) I think helpers on `Standard` is the way to go :+1:

Reply:
I've added helpers on `CmdError`